packages/drawer/src/resources/drawer-util.ts

```diff
@@ -0,0 +1,21 @@
+import { UxDrawer } from '../ux-drawer';
+
+export function findDrawer(item: HTMLElement | null): UxDrawer | null {
+  let element = item;
+  if (element === null) return null;
+  while (element.tagName !== 'BODY' && element.tagName !== 'UX-DRAWER') {
+    element = element.parentElement;
+    if (element === null) return null;
+  }
+  if (element.tagName === 'UX-DRAWER') {
+    const el: any = element;
+    if (
+      el !== null &&
```

Comment:
This is personal preference: I think operator should be at the start of the next line. The reason is we shouldn't have to read the entire line to see the relationship of 2 lines